### DISCUSSION TITLE
How can we create form UI

### DISCUSSION THREAD

**Discussion by menoc61** [founded-labs/react-native-reusables#143] [Q&A]
Hey 👋, I'm a great fan of your work and i have stated creating an expo application using these components. It was quite easy and intuitive until a stuck on creating the authentication screens, I wanted to use the Form UI but it seems to be depricated and some to it's elements relies on depricated elements, I looked at the documentation and there is nothing about it, what should I do ?
I want to make this library better even with the documentation by making previews on native devices but after finishing with my side project... Please I need help for that.

**Comment by mrzachnugent** [comment]
Hey @menoc61,

The Form UI is on my  `// TODO` list. You can follow the issue for updates and I left a comment for the current implementation from the showcase app https://github.com/mrzachnugent/react-native-reusables/issues/104#issuecomment-2142772112. The Form Components are deprecated so proceed with caution.
